chart.js/index.d.ts

```diff
@@ -3,15 +3,15 @@
 // Definitions by: Alberto Nuti <https://github.com/anuti>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare enum ChartType {
+export enum ChartType {
     line, bar, radar, doughnut, polarArea, bubble
 }
-declare enum TimeUnit {
+export enum TimeUnit {
```

Comment:
Also possibly prior to your changes, but Chart.js also does not export a value named `TimeUnit`, and `TimeScale` (the only case where `TimeUnit` is used) only allows strings per the documentation ([here](https://github.com/chartjs/Chart.js/blob/7c8998f96adeece76000406ca06ec8bb178392d7/docs/02-Scales.md#time-scale) and [here](https://github.com/chartjs/Chart.js/blob/7c8998f96adeece76000406ca06ec8bb178392d7/docs/02-Scales.md#time-units)). Perhaps this should be a type alias:

```ts
export declare type TimeUnit = "millisecond" | "second" | "minute" | "hour" | "day" | "week" | "month" | "quarter" | "year";
```